kroxylicious-api/src/main/java/io/kroxylicious/proxy/filter/FilterDispatchExecutor.java

```diff
@@ -44,6 +44,7 @@ public interface FilterDispatchExecutor extends ScheduledExecutorService {
      * @return a stage completed (both exceptionally and normally) by this Filter Dispatch Thread
      * @param <T> stage value type
      */
-    <T> CompletionStage<T> completeOnFilterDispatchThread(@NonNull CompletionStage<T> stage);
+    <T> CompletionStage<T> completeOnFilterDispatchThread(@NonNull
```

Comment:
annotated parameters look a bit funky

Reply:
I can make intelliJ behave but the eclipse formatter is not happy

Reply:
https://youtrack.jetbrains.com/issue/IJPL-39084/Eclipse-code-style-import-Line-Wrapping-improve-single-setting-processing seems related.